### DISCUSSION TITLE
Welcome to WikiLabs "plugins" Discussions Group

### DISCUSSION THREAD

**Discussion by pmario** [wikilabs/plugins#50] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kookma** [comment]
@pmario - wikilabs is one of best TW resource on the web. It uses a nice workflow for creating editions and plugins and includes several useful tools.

**Reply by pmario** [reply]
Thx!
